doc/arch/adr-004-geoprocessing-api.md

```diff
@@ -0,0 +1,50 @@
+## 004 - Expose Geoprocessing API
+
+### Context
+The MMW/BigCZ platform performs a number of geospatial processing tasks which can be initiated by submitting tasks to an HTTP endpoint.  The web application provides the tools and workflow for assembling and submitting the input, monitoring execution progress, and collecting and visualizing the results of these jobs.  The platform provides value by assembling the required national scale data and the infrastructure necessary to perform the processing. In an effort to extend this capability to alternate workflows that don't involve the web platform, the client wants to formalize an externally consumable HTTP API.
+
+#### API Objectives
+* Support execution of all current and future "Analyze" sub tasks (Land Use, Soil, etc)
+* Support execution of Rapid Watershed Delineation
+* No model execution will be possible
```

Comment:
Is model execution actually not possible, or do you mean that we don't have to make it possible? Consider rephrasing to "Will not support model execution"